content/blog/blog-post-09.md

```diff
@@ -0,0 +1,37 @@
+---
+authors: ["kokkos-team"]
+title: "Kokkos Releases New Autotuning Features"
+date: 2024-12-10
+tags: ["blog"]
+thumbnail: img/blog/2024/apex-kokkos.jpg
+---
+
+# Motivation
+
+By default, internal Kokkos execution space parameters are empirically or heuristically hand-tuned with fixed parameter values to provide "one size fits most" performance, with the goal of minimizing the effect of the abstraction overhead and approximating the performance of an optimized, lower-level implementation. Can these parameters be tuned for a particular application and architecture for programmers to easily tackle further performance opportunities? The Kokkos Tools APEX auto-tuning connector [5], a git submodule in Kokkos Tools with a stable version released in Kokkos 4.5 [4], offers an answer.
```

Comment:
Is the git-submodule part important?  I would not mention it.
Also I don't quite get what you mean by "a stable version released in 4.5".  There is currently still no Kokkos Tools releases in lock step with Kokkos and Kokkos Kernels releases.